### ISSUE TITLE
Crash with create:big cannons mod

### ISSUE THREAD

**Issue by foreverStarship** [LtxProgrammer/Changed-Minecraft-Mod#386]
*Translator warning
Probably a long time ago, when I ran the changed mod with create:big cannons and it crashed during the startup game phase, I thought it was a rendering issue.
Here's the crash report:
[crash-2023-02-24_21.03.03-client.txt](https://github.com/LtxProgrammer/Changed-Minecraft-Mod/files/11128847/crash-2023-02-24_21.03.03-client.txt)

Game version:1.18.2 Forge_40.2.1;
RAM:I think I allocated 8G;
Changed mod version:v0.10.2;
Create:Big Cannons version:beta-0.5.b;

Ps:Chinese warning if you want to view the crash report
Pss:I don't think this is a good question but I think it probably means that you'll meet this question with other mods because of rendering

**Comment by LtxProgrammer** [comment]
Try running the mod with the latest version of changed.

**Comment by foreverStarship** [comment]
Solved,but.........I think I found another issue?(should I close this a add a new issue?)

Mod lists:
Changed-m1.18.2-v0.11c
create-1.18.2-0.5.0.i
flywheel-forge-1.18.2-0.6.8
i18nupdatemod-1.18-3.0.2 (just a auto translate mod)

Problems:
I create a new world and after a few seconds I crashed.
And I don't know why I don't have a crash report.
Here's the lastest log:
[latest.log](https://github.com/LtxProgrammer/Changed-Minecraft-Mod/files/11129586/latest.log)

Solved?
No.

How to reproduce?
Add the mods and create a new world,then wait for a few sec.
